qiskit/providers/aer/backends/aer_simulator.py

```diff
@@ -535,9 +531,15 @@ def from_backend(cls, backend, **options):
 
         # Use automatic noise model if none is provided
         if 'noise_model' not in options:
-            noise_model = NoiseModel.from_backend(backend)
-            if not noise_model.is_ideal():
-                options['noise_model'] = noise_model
+            # pylint: disable=import-outside-toplevel
+            # Avoid cyclic import
+            from ..noise.noise_model import NoiseModel
+
+            if 'delay_noise' in options:
```

Comment:
I dont think this is necessary here. If you want to do custom configuration of the noise model you should do `noise_model=NoiseModel.from_backend(backend, **custom_noise_options)`.

It is strange to only have handling for one of the noise model options here.

Reply:
Done at 053130f

Reply:
I also removed `delay_noise` option from `NoiseModel.from_backend` (e959be86).
We now have no public way to replicate old behavior but we still have a private way with one additional line for it:
```
noise_model = NoiseModel.from_backend(backend)
noise_model._custom_noise_passes = []	 # a bit hack though
```